src/main/java/LacksDataBaseProject/AddAndRemoveLack.java

```diff
@@ -0,0 +1,53 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveLack extends CheckUserRole {
+    static Scanner input = new Scanner(System.in);
```

Comment:
trzymanie takich staticów jak Scanner jest dość niebezpiecznie, bo łatwo o memory leaky. Generalnie na scannerze powinno sie robić close bo zakończeniu działania, a dodatkowo jeśli jest statyczny to bez względu na instancje obiektu i tak jest trzymany. Dodatkowo, pewnie będziecie chcieli potem zrobić jakieś gui do apki, a scanner nie jest tam wykorzystywany, więc wydaje mi się, że najlepiej by było napisać klasy tak by zależności do tego scannera wogóle nie było i dodatkowo do klasy powinny być przekazywane jakies argumenty końcowe pobrane za pomocą scanner czy gui, tak by można było łatwo to zintegrować :) wiem gruby task, ale zastanówcie się czy nie możecie jakiegoś mostu tutaj użyć, albo odseparować od siebie elementy.